pkg/backend/apply.go

```diff
@@ -235,6 +237,9 @@ func PreviewThenPromptThenExecute(ctx context.Context, kind apitype.UpdateKind,
 		DryRun:   false,
 		ShowLink: true,
 	}
+	// No need to generate a plan at this stage, there's no way for the system or user to extract the plan
+	// after here.
+	op.Opts.Engine.GeneratePlan = false
```

Comment:
This feels a bit ick, but the nature of preview and update (and everything else really) running through the same bit of the engine results in this looking like the most sensible place to toggle this.